charts/greptimedb-standalone/templates/configmap.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if .Values.configToml -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-config
+  namespace: {{ .Release.Namespace }}
```

Comment:
suggestion: Add helm hook like [this](https://helm.sh/docs/topics/charts_hooks/)，configmap need 
`pre-install` ？


<br class="Apple-interchange-newline">

Reply:
It seems complicated. I'm not sure we should do that.